docs/howto/exitcode.adoc

```diff
@@ -0,0 +1,61 @@
+= How to use exit codes in qDup
+
+qDup is designed to automate the steps a user takes when working in an ssh terminal session. Users normally check command output messages and not exit codes.
+
+For example, if I want to run a bash script `./missing.sh` I look for the `No such file or directory` error message instead of checking that the exit code with `echo $?`.
+
+```
+#> ./missing.sh
+bash: ./missing.sh: No such file or directory
+#> echo $?
+127
+```
+There are, however, times when a commands exit code provides important information. In those cases, we have two ways to work with exit codes.
```

Comment:
I wouldn't say "two ways to work with exit codes" because these are not really 'generic' ways. Rather sum it up as "abort on non-zero" and "process exit code as text" or something similar.